### DISCUSSION TITLE
EvilMartians post using Phlex on app

### DISCUSSION THREAD

**Discussion by megatux** [yippee-fun/phlex#540] [Show and tell]
Probably old news, but anyway, I saw on the ruby weekly the link to this post from EvilMartians of an example app [using Phlex](https://evilmartians.com/chronicles/anycable-goes-off-rails-connecting-twilio-streams-with-hanami#using-phlex-for-views) and tons of other cool stuff.

**Comment by joeldrapper** [comment]
Thanks for highlighting this. I think that's the first blog post mentioning Phlex.
